lib/src/badge/types.ts

```diff
@@ -0,0 +1,5 @@
+type Props = {
+  notificationText?: boolean | number;
```

Comment:
I think it should be also `string` type, as in the tabs (where the badge is being used), if the notificationNumber is > 99, `+99` will be put as notificationText in the badge.